Runtime/EdgegapServerBootstrap.cs

```diff
@@ -0,0 +1,85 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Edgegap.Bootstrap
+{
+    public class EdgegapServerBootstrap : MonoBehaviour
+    {
+        public static EdgegapServerBootstrap Instance { get; protected set; }
+        public static string BootstrapObjectName = "EdgegapServerBootstrap";
+
+        protected const string APP_VERSION_PAGE = "https://app.edgegap.com/application-management/applications/list";
+
+        protected ArbitriumPortsMapping _arbitriumPortsMapping;
+        protected ArbitriumDeploymentLocation _arbitriumDeploymentLocation;
+        protected Dictionary<string, string> _arbitriumSimpleEnvs = new Dictionary<string, string>();
+
+        protected enum EdgegapTransportProtocols
+        {
+            TCP,
+            UDP,
+            HTTP,
+            HTTPS,
+            TCP_UDP,
+            WS,
+            WSS
+        }
+
+        private void Awake()
+        {
+            if (Instance != null && Instance != this)
+            {
+                Destroy(this);
+            }
+            else
+            {
+                Instance = this;
+            }
+        }
+
+        private void Start()
+        {
+#if !UNITY_CLIENT && !UNITY_EDITOR
+            ParseEdgegapEnvs();
+            ValidatePortMapping();
+#endif
+        }
+
+        private void ParseEdgegapEnvs()
+        {
+            IDictionary envs = Environment.GetEnvironmentVariables();
+
+            foreach (DictionaryEntry envEntry in envs)
+            {
+                if (envEntry.Key.ToString().Contains("ARBITRIUM_"))
```

Comment:
use early exit pattern, flip condition to `!envEntry...Contains(...) { continue; }` to remove a level of indentation and improve readability